cc/mcts_player.cc

```diff
@@ -164,7 +165,7 @@ Coord MctsPlayer::SuggestMove() {
 }
 
 Coord MctsPlayer::PickMove() {
-  if (!options_.soft_pick || root_->position.n() >= temperature_cutoff_) {
```

Comment:
with >=, doesn't that mean 30 is the first argmax?   i.e., black has one extra softpick move?  or is n indexed from 0?

Reply:
I 99% know it's indexed from 0 in Python. I assume it is in C++ too.

Reply:
>= is correct, it was originally incorrectly >

See PR #213